content/docs/user-guide/basic-concepts/workspace.md

```diff
@@ -6,3 +6,39 @@ tooltip: >-
   models, etc. Typically, it's also a Git repository. It will contain your DVC
   project.
 ---
+
+# Workspace
+
+A data science project consists of data obtained from many different sources.
+This data may be split into multiple files or directories or (as the project
+structure needs) have different versions for different requirements. e.g. A
+smaller / simplified version might be required in prototyping for faster
+feedback and shorter training times. A single workspace to manage all artifacts
+of a project is desirable, although versioning needs and managing dependencies
+make it increasingly complex.
+
+DVC allows a single directory to contain all your project artifacts. The
+workspace is the directory containing _user visible_ part of your
+<abbr>project</abbr> e.g. raw datasets, source code, ML models, etc. Users work
+in this directory using their data and model files that and manipulate the
+contents through DVC commands.
+
+Files and directories in the workspace can be added to DVC (`dvc add`) or they
+can be downloaded from external sources (`dvc get`, `dvc import`,
+`dvc import-url`). Changes to the files, directories, notebooks, models, and in
+general machine learning file system can be tracked (`dvc commit`) and versioned
+in Git (`dvc checkout`). They can be removed (`dvc remove`) from the workspace.
+<abbr>Pipelines</abbr> and <abbr>dependencies</abbr> between them can be
+defined. Data and model files can be moved to the cloud and retrieved when
+necessary (`dvc push`, `dvc pull`). DVC supports all typical operations of files
+and directories of a file system through its commands.
```

Comment:
> DVC supports all typical operations...

What do we mean by this?

Reply:
We mean the user can live in the workspace and use dvc to do what they normally do with files. Create, copy, rename etc.

Reply:
I see. Yes it may be a relevant note. This sentence seems to belong more to the previous paragraph though? Something like:

"There's usually no need to modify tracked data manually as DVC commands provide ways to do so, but if you do, use `dvc commit` to register the changes..."

(see my previous comment, these would have to be merged somehow).